src/deprecated.jl

```diff
@@ -357,3 +357,32 @@ function by(d::AbstractDataFrame, cols::Any; sort::Bool=false, skipmissing::Bool
 end
 
 @deprecate groupvars(gd::GroupedDataFrame) groupcols(gd)
+
+@deprecate aggregate(d::AbstractDataFrame, fs::Any; sort::Bool=false) begin
```

Comment:
To make the deprecation a bit simpler, it would be nice to branch on `sort` and use `depwarn` to print a deprecation warning which doesn't use `sort` when it's not provided (using `sort=nothing` by default).

Reply:
OK - I made two branches.